### PR TITLE
Added number type to output of IsStrongPassword

### PR DESCRIPTION
**Pull request by roceb**
Please fill in this template.

- [ x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ x ] Test the change in your own code. (Compile and run.)
- [ x ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [ x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ x ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ X ] Provide a URL to documentation or source code which provides context for the suggested changes: [Url](https://github.com/validatorjs/validator.js#validators)
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/validator/index.d.ts [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ declare namespace validator {
         pointsForContainingNumber?: number | undefined;
         pointsForContainingSymbol?: number | undefined;
     }
-    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean;
+    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean| number;
```
Comment (peterblazejewicz) [inline types/validator/index.d.ts:1066]:
```suggestion
    function isStrongPassword(str: string, options?: strongPasswordOptions): boolean;
    function isStrongPassword(str: string, options: strongPasswordOptions & { returnScore: true): number;
```
should this be something more like above?
https://github.com/validatorjs/validator.js/blob/63b61629187a732c3b3c8d89fe4cacad890cad99/src/lib/isStrongPassword.js#L89

File: types/validator/validator-tests.ts [hunk 2]
```diff
@@ -924,6 +922,11 @@ const any: any = null;
     result = validator.toInt(any);
     result = validator.toInt(any, 10);
 }
+{
+    let result: boolean|number;
+    result = validator.isStrongPassword("sample");
+    result = validator.isStrongPassword("sample23#@test");
```
Comment (peterblazejewicz) [inline types/validator/validator-tests.ts:928]: this is not correct IMO, see my comment about return type. This require scoring function switch IMO.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
+1 to the other comments

**Comment by jjdive** [comment]
Just created a new PR with an attempt on fixing this: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61951

**Comment by peterblazejewicz** [comment]
@jjdive please close this one as no longer relevant

**Comment by peterblazejewicz** [comment]
Closing as not relevant after #61951

**Pull request closed without merging**